thirdparty/CMakeLists.txt

```diff
@@ -14,4 +14,35 @@
 # limitations under the License.
 # ******************************************************************************
 
+if(ENABLE_PROFILING_ITT)
```

Comment:
maybe a separate directory `thirdparty/ittnotify/CMakeLists.txt`?

Reply:
Do you mean  to. add ittnotify as a submodule or just create a folder with cmake script?

Reply:
> just create a folder with cmake script?

yes. `thirdparty` just a root cmake where only `add_subdirectory` calls.

Reply:
Ok, fixed